syncd/tests/TestSyncdMlnx.cpp

```diff
@@ -78,7 +78,7 @@ void syncdMlnxWorkerThread()
     commandLineOptions->m_enableTempView = false;
     commandLineOptions->m_disableExitSleep = true;
     commandLineOptions->m_enableUnittests = false;
-    commandLineOptions->m_enableSaiBulkSupport = true;
+    commandLineOptions->m_enableSaiBulkSupport = false;
```

Comment:
why this change ?

Reply:
just to leverage the pipeline to check whether coverage can pass in this way. we will have a formal solution if it can.
it's not easy for us to check coverage locally.

Reply:
you can actually do it in docker, it  will cost you some time to setup this locally, and then you can run gcovr following steps taht are located in *yml files

i don;t have this steps locally, but i usually write unittests for code that i add/change

Reply:
thanks. indeed. we usually toggle breakpoints and check whether breakpoints are reached.
but sometimes even if the breakpoints are reached, coverage reports uncovered..
we will try steps in yaml.